doc/memory.rst

```diff
@@ -60,6 +60,13 @@ A simple example:
     Running f(2)
     2
 
+.. note::
```

Comment:
Would it be possible to make it explicit that this note is actually a security warning?

```suggestion
.. warning::
```